tests/unit/test_workflow_files.py

```diff
@@ -307,14 +306,13 @@ def test_parse_reg__ok(
         expected_path: Expected path returned for src=True only.
     """
     paths: dict = setup__test_parse_reg()
-    expected: Union[str, Tuple[str, Path]]
+    expected: Tuple[str, Path]
     expected_reg = expected_reg.format(**paths)
     if src:
         expected_path = expected_path.format(**paths)
-        expected = (expected_reg, Path(expected_path))
+        assert Path(expected_path) == parse_reg(reg, src)[1]
     else:
-        expected = expected_reg
-    assert parse_reg(reg, src) == expected
```

Comment:
I have opened a PR against this PR with suggestions for these tests - https://github.com/hjoliver/cylc-flow/pull/18

Reply:
Merged, thanks